docs/my-website/docs/providers/snowflake.md

```diff
@@ -0,0 +1,101 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Snowflake
+LiteLLM supports all Snowflake models.
+- `snowflake/deepseek-r1`
```

Comment:
I don't think the docs needs a list like this, snowflake can change / update this list